python3/koans/about_modules.py

```diff
@@ -45,21 +47,21 @@ def test_importing_all_module_attributes_at_once(self):
         self.assertEqual(__, hamster.name)
 
     def test_modules_hide_attributes_prefixed_by_underscores(self):
-        with self.assertRaises(___): private_squirrel = _SecretSquirrel()
+        with self.assertRaises(___):
+            private_squirrel = _SecretSquirrel()
 
     def test_private_attributes_are_still_accessible_in_modules(self):
-        from .local_module import Duck # local_module.py
+        from .local_module import Duck  # local_module.py
 
         duck = Duck()
         self.assertEqual(__, duck._password)
         # module level attribute hiding doesn't affect class attributes
         # (unless the class itself is hidden).
 
-    def test_a_module_can_choose_which_attributes_are_available_to_wildcards(self):
-        # NOTE Using this module level import declared at the top of this script:
-        #   from .local_module_with_all_defined import *
+    def test_a_modules_XallX_statement_limits_what_wildcards_will_match(self):
```

Comment:
The name is throwing me off a little here... why `_XallX_`?  I find it a little hard to read, and it's actually `__all__` as well.

Maybe `test_a_module_can_limit_what_is_imported_when_importing_everything` or `_with_a_wildcard`?